framework/include/ble_manager/ble_client.h

```diff
@@ -83,6 +83,7 @@ typedef struct {
 typedef struct {
 	uint8_t raw_data[BLE_ADV_RAW_DATA_MAX_LEN];
 	uint8_t raw_data_length;
+	uint8_t mac[BLE_BD_ADDR_MAX_LEN];
```

Comment:
When will this be used?

Reply:
This is a format for implementing BLE driver.
After 'BLE drirver' works done, I will make a new example.